metadata-ingestion/src/datahub/ingestion/source/sql/presto_on_hive.py

```diff
@@ -129,6 +129,16 @@ class PrestoOnHiveConfig(BasicSQLAlchemyConfig):
         description="Add the Presto catalog name (e.g. hive) to the generated dataset urns. `urn:li:dataset:(urn:li:dataPlatform:hive,hive.user.logging_events,PROD)` versus `urn:li:dataset:(urn:li:dataPlatform:hive,user.logging_events,PROD)`",
     )
 
+    extra_properties: List[str] = Field(
+        default=[],
+        description="By default, the connector extracts a specific set of properties from the metastore tables. Use this list to provide additional properties that you would like to extract.",
```

Comment:
We need to make this clearer that this is the list of *keys*, and the values are extracted at runtime